src/dune

```diff
@@ -7,4 +7,4 @@
   (backend bisect_ppx))
  (flags
   (:standard -w -50))
- (libraries astring result camlp-streams))
+ (libraries result camlp-streams))
```

Comment:
Why is removing `astring` useful while Odoc uses it too ?

Reply:
odoc-parser is used by more than just odoc. In this particular case, since once we drop support for <=4.03 the functionality we need is entirely in stdlib, it seems just good hygiene. I'm quite keen to drop old OCaml support fairly soon (and at a major version number).

Reply:
Why wait?

Reply:
You are right, with the planned removal of the new compat functions, I don't see a problem with adding them.